packages/appium/lib/main.js

```diff
@@ -170,6 +172,18 @@ function areServerCommandArgs(args) {
 async function init(args) {
   const appiumHome = args?.appiumHome ?? (await resolveAppiumHome());
 
+  // this is here so extensions installed in e.g., `~/.appium` can find the peer dependency
+  // of `appium`, which lives wherever it lives (see `rootDir`).
+  if (!(await hasAppiumDependency(appiumHome))) {
+    try {
+      await fs.symlink(rootDir, path.join(appiumHome, 'node_modules', 'appium'));
```

Comment:
would this work on Windows?

Reply:
I assume we need to use junction there

Reply:
I assume we need to call mkdirp first to create `node_modules` root in `appiumHome`

Reply:
are you sure we need `junction`?  I thought that was a hard link--e.g., if `~/.appium/node_modules/appium` is deleted then e.g., `$NPM_PREFIX/node_modules/appium` also gets deleted?

Reply:
on Windows - yes. it cannot create links for folders. Check https://github.com/nodejs/node/issues/18518, but you could try it yourself. Btw I know a company, which provides a whole VM cloud, where Windows-related stuff could also be tested ;)